Build errors on a Fresh clone

I cloned the repo fresh and ran `cabal build` but was greeted with these errors


app/Main.hs:58:5: error:
    • Couldn't match expected type: IO ()
                  with actual type: D.DataFrame -> IO ()
    • Probable cause: ‘D.plotHistograms’ is applied to too few arguments
      In a stmt of a 'do' block: D.plotHistograms parsed
      In the expression:
        do parsed <- D.readCsv "./data/housing.csv"
           mapM_
             (print . (\ name -> (name, D.columnSize name parsed)))
             (D.columnNames parsed)
           print $ D.take 5 parsed
           D.plotHistograms parsed
      In an equation for ‘housing’:
          housing
            = do parsed <- D.readCsv "./data/housing.csv"
                 mapM_
                   (print . (\ name -> (name, D.columnSize name parsed)))
                   (D.columnNames parsed)
                 print $ D.take 5 parsed
                 ....
   |
58 |     D.plotHistograms parsed
   |     ^^^^^^^^^^^^^^^^^^^^^^^

app/Main.hs:58:22: error:
    • Couldn't match expected type ‘D.HistogramOrientation’
                  with actual type ‘D.DataFrame’
    • In the first argument of ‘D.plotHistograms’, namely ‘parsed’
      In a stmt of a 'do' block: D.plotHistograms parsed
      In the expression:
        do parsed <- D.readCsv "./data/housing.csv"
           mapM_
             (print . (\ name -> (name, D.columnSize name parsed)))
             (D.columnNames parsed)
           print $ D.take 5 parsed
           D.plotHistograms parsed
   |
58 |     D.plotHistograms parsed
   |                      ^^^^^^

app/Main.hs:66:5: error:
    • Couldn't match expected type: IO a0
                  with actual type: D.DataFrame -> IO ()
    • Probable cause: ‘D.plotHistograms’ is applied to too few arguments
      In a stmt of a 'do' block: D.plotHistograms rawFrame
      In the expression:
        do rawFrame <- D.readCsv
                         "./data/effects-of-covid-19-on-trade-at-15-december-2021-provisional.csv"
           print $ dimensions rawFrame
           print $ D.take 10 rawFrame
           D.plotHistograms rawFrame
           ....
      In an equation for ‘covid’:
          covid
            = do rawFrame <- D.readCsv
                               "./data/effects-of-covid-19-on-trade-at-15-december-2021-provisional.csv"
                 print $ dimensions rawFrame
                 print $ D.take 10 rawFrame
                 ....
   |
66 |     D.plotHistograms rawFrame
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^

app/Main.hs:66:22: error:
    • Couldn't match expected type ‘D.HistogramOrientation’
                  with actual type ‘D.DataFrame’
    • In the first argument of ‘D.plotHistograms’, namely ‘rawFrame’
      In a stmt of a 'do' block: D.plotHistograms rawFrame
      In the expression:
        do rawFrame <- D.readCsv
                         "./data/effects-of-covid-19-on-trade-at-15-december-2021-provisional.csv"
           print $ dimensions rawFrame
           print $ D.take 10 rawFrame
           D.plotHistograms rawFrame
           ....
   |
66 |     D.plotHistograms rawFrame


Perhaps I did something wrong? I was able to fix this Ill create a PR with the fix I have. 

I think the README would benefit with some instructions on how to use it.

But regardless I love what I see so far. Let me know if you would like me to add some nix support.

--- a/app/Main.hs
+++ b/app/Main.hs
@@ -55,7 +55,7 @@ housing = do
 
     print $ D.take 5 parsed
 
-    D.plotHistograms VerticalHistogram parsed
+    D.plotHistograms D.VerticalHistogram parsed
 
 covid :: IO ()
 covid = do
@@ -63,7 +63,7 @@ covid = do
     print $ dimensions rawFrame
     print $ D.take 10 rawFrame
 
-    D.plotHistograms VerticalHistogram rawFrame
+    D.plotHistograms D.VerticalHistogram rawFrame
 
     -- value of all exports from 2015
     print $ rawFrame

--- a/app/Main.hs
+++ b/app/Main.hs
@@ -55,7 +55,7 @@ housing = do
 
     print $ D.take 5 parsed
 
-    D.plotHistograms parsed
+    D.plotHistograms VerticalHistogram parsed
 
 covid :: IO ()
 covid = do
@@ -63,7 +63,7 @@ covid = do
     print $ dimensions rawFrame
     print $ D.take 10 rawFrame
 
-    D.plotHistograms rawFrame
+    D.plotHistograms VerticalHistogram rawFrame
 
     -- value of all exports from 2015
     print $ rawFrame

